components/validation/test/validation-custom.test.js

```diff
@@ -0,0 +1,70 @@
+import '../validation-custom.js';
+import { aTimeout, expect, fixture, html, oneEvent } from '@open-wc/testing';
+import { runConstructor } from '../../../tools/constructor-test-helper.js';
+
+const basicFixture = html`
+	<div>
+		<div>
+			<input id="target" type="text"/>
+		</div>
+		<d2l-validation-custom for="target" failure-text="An error occurred"></d2l-validation-custom>
+	</div>
+`;
+
+describe('d2l-validation-custom', () => {
+
+	let root, custom;
+
+	beforeEach(async() => {
+		root = await fixture(basicFixture);
+		custom = root.querySelector('d2l-validation-custom');
+	});
+
+	describe('constructor', () => {
+
+		it('should construct', () => {
+			runConstructor('d2l-validation-custom');
+		});
+
+	});
+
+	describe('target', () => {
+
+		it('should find target using for attribute', () => {
+			const expectedTarget = root.querySelector('#target');
+			expect(custom.target).to.equal(expectedTarget);
+		});
+
+	});
+
+	describe('events', () => {
+
+		it('should fire connected event', async() => {
+			const custom = document.createElement('d2l-validation-custom');
```

Comment:
Careful with scopes here... there's already a `custom` defined above so this initially confused me.